packages/flutter/test/widgets/form_test.dart

```diff
@@ -1126,4 +1126,174 @@ void main() {
+
+    // Enter text in the first TextFormField.
+    await tester.enterText(find.byType(TextFormField).first, 'foo');
+    await tester.pumpAndSettle();
+
+    // No error text is visible yet.
+    expect(find.text(errorText('foo')!), findsNothing);
+
+    // Tap on the second TextFormField to trigger validation.
+    await tester.tap(find.byType(TextFormField).last);
+    await tester.pumpAndSettle();
+
+    // Tap on the first TextFormField to trigger validation.
+    await tester.tap(find.byType(TextFormField).first);
+    await tester.pumpAndSettle();
+
+    // Verify that the both error texts are displayed.
+    expect(find.text(errorText('foo')!), findsOneWidget);
+    expect(find.text(errorText('bar')!), findsOneWidget);
+  });
+
+  testWidgets('Validate conflicting AutovalidateModes', (WidgetTester tester) async {
+    final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+    String? errorText(String? value) => '$value/error';
+
+    Widget builder() {
+      return MaterialApp(
+        theme: ThemeData(),
+        home: MediaQuery(
+          data: const MediaQueryData(),
+          child: Directionality(
+            textDirection: TextDirection.ltr,
+            child: Center(
+              child: Form(
+                key: formKey,
+                autovalidateMode: AutovalidateMode.onUnfocus,
+                child: Material(
+                  child: Column(
+                    children: <Widget>[
+                      TextFormField(
+                        autovalidateMode: AutovalidateMode.always,
+                        initialValue: 'bar',
+                        validator: errorText,
+                      ),
+                      TextFormField(
+                        autovalidateMode: AutovalidateMode.disabled,
+                        initialValue: 'bar',
+                        validator: errorText,
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(builder());
+
+    // Verify that the error text is displayed for the first TextFormField.
+    expect(find.text(errorText('bar')!), findsOneWidget);
+
+    // Enter text in the TextFormField.
+    await tester.enterText(find.byType(TextFormField).first, 'foo');
+    await tester.pumpAndSettle();
+
+    //click in the second TextFormField to trigger validation
+    await tester.tap(find.byType(TextFormField).last);
+    await tester.pumpAndSettle();
+
+    // No error text is visible yet for the second TextFormField.
+    expect(find.text(errorText('bar')!), findsNothing);
+
+    // Now click in the first TextFormField to trigger validation for the second TextFormField.
+    await tester.tap(find.byType(TextFormField).first);
+    await tester.pumpAndSettle();
+
+    // Verify that the error text is displayed for the second TextFormField.
+    expect(find.text(errorText('bar')!), findsOneWidget);
+  });
```

Comment:
So the behavior in this case is that the first TextField respects its AutovalidateMode.always, but the second TextField doesn't respect AutovalidateMode.disabled, and instead uses the Form's AutovalidateMode.onUnfocus? Is that typical behavior for other values of AutovalidateMode?

Reply:
It will just ignore the AutovalidateMode.disabled